### PR TITLE
Create Documented Context API properties

### PR DESCRIPTION
**Pull request by blackswanny**
add and update manual document for Context API based on Koa Context

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -331,55 +370,53 @@ In the server, `ctx` also exposes the same properties as a [Koa context](http://
...
-    * `properties: Object` - is merged to the error object
-  * `respond: boolean` - set to true to bypass Koa's built-in response handling. You should not use this flag.
+  * `acceptsEncodings: (...encodings: ...string) => boolean` - check if encodings are acceptable
+  * `acceptsCharset: (...charsets: ...string) => boolean` - check if charsets are acceptable
+  * `acceptsLanguages: (...languages: ...string) => boolean` - check if langs are acceptable
+  * `get: (name: String) => string` - returns a header field
+  
+  </details>
+  
+  <details><summary>Properties below are aliases of Koa's response object</summary>
```
Comment (KevinGrandon) [inline README.md]: It seems not very clear to me that these are properties under the `response` key. Can we add a bullet point here /w key name still? I'm not sure if this works, but something like:

```md
* `response: Object`: <details><summary>View koa response details</summary>...
```

Comment (blackswanny) [reply README.md]: changed

File: README.md [hunk 2]
```diff
@@ -322,7 +322,47 @@ When a request does not require a server-side render, `ctx.body` follows regular
...
+  * `assert: (value: any, status: ?number, message: ?string, properties: ?Object)` - throws if `value` is falsy. Uses [Assert](https://github.com/jshttp/http-assert)
+    * `value: any`
+    * `status: number` - HTTP status code
+    * `message: string` - error message
+    * `properties: Object` - is merged to the error object
+  * `respond: boolean` - set to true to bypass Koa's built-in response handling. You should not use this flag.
+  * `app: Object` - a reference to the Koa instance
+  * `cookies: {get, set}` - cookies based on [Cookie Module](https://github.com/pillarjs/cookies)
+  
+  <details><summary>View Koa cookies details</summary>
```
Comment (KevinGrandon) [inline README.md]: I still think we want something that signifies that we access these properties through the `cookies/response/request` keys.  Does something like this work?

```
* `response: Object`: <details>...
```

**Comment by blackswanny** [comment]
cc @lhorie @nadiia @AlexMSmithCA

**Comment by blackswanny** [comment]
!merge

**Pull request merged**